### PR TITLE
Update installing tutorial to csproj

### PR DESCRIPTION
**Pull request by deakgeek**
#96

### CODE REVIEW

File: content/efcore/getting-started/installing-ef-core.md [hunk 1]
```diff
@@ -1,58 +1,80 @@
-# Installing EF Core  
+# Installing EF Core 
  
-In this lesson, you will learn how to install Entity Framework Core (EF Core) on your machine.
+In this lesson, you will learn how to install Entity Framework Core on your machine.
  
-## Install .NET Core 
+## Install .NET Core
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: Remove extra whitespace

File: content/efcore/getting-started/installing-ef-core.md [hunk 2]
```diff
@@ -1,58 +1,80 @@
-# Installing EF Core  
+# Installing EF Core 
  
-In this lesson, you will learn how to install Entity Framework Core (EF Core) on your machine.
+In this lesson, you will learn how to install Entity Framework Core on your machine.
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: We reference it as EF Core later (heading), so we still need to specify the acronymn.

File: content/efcore/getting-started/installing-ef-core.md [hunk 3]
```diff
@@ -1,58 +1,80 @@
...
+  <ItemGroup>  
+    <DotNetCliToolReference  
+    Include="Microsoft.EntityFrameworkCore.Tools.DotNet"   
+    Version="1.0.0" />  
+  </ItemGroup> 
+</Project> 
+```  
  
-After running this command, EF Core should be installed on your machine. You can run the following command to ensure EF Core was installed correctly: 
+Once you have updated the `csproj` file, run the following command in your project directory to install your project's tools and dependencies:
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: "in the project directory"

**Review by helmoski — CHANGES_REQUESTED** [review]
Lessons that reference this tutorial should be updated to reflect the fact that it is now an appendix.

File: content/efcore/getting-started/installing-ef-core.md (hunk 3 above)
Comment (deakgeek) [reply content/efcore/getting-started/installing-ef-core.md]: Done

File: content/efcore/getting-started/installing-ef-core.md (hunk 1 above)
Comment (deakgeek) [reply content/efcore/getting-started/installing-ef-core.md]: Done

File: content/efcore/getting-started/installing-ef-core.md (hunk 2 above)
Comment (deakgeek) [reply content/efcore/getting-started/installing-ef-core.md]: Done

File: content/efcore/getting-started/generating-databases-and-data-models.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ dotnet ef migrations add [migration name]
 ``` 
  
 > **Note** {.note}  
-> If the above command fails with the error message, `No executable found matching command "dotnet-ef"`, make sure `Microsoft.EntityFrameworkCore.Tools` has been installed for your project, as shown in the [Installing EF Core](installing-ef-core.md) lesson. Also ensure you are running the command from the directory that contains your `project.json` file. 
+> If the above command fails with the error message, `No executable found matching command "dotnet-ef"`, make sure `Microsoft.EntityFrameworkCore.Tools` has been installed for your project, as shown in the [Installing EF Core](installing-ef-core.md) appendix. Also ensure you are running the command from the directory that contains your `project.json` file.
```
Comment (helmoski) [inline content/efcore/getting-started/generating-databases-and-data-models.md:16]: It is also referenced in the "Generating a Data Model from an Existing Database" section

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/generating-databases-and-data-models.md (hunk 4 above)
Comment (deakgeek) [reply content/efcore/getting-started/generating-databases-and-data-models.md:16]: Done

**Review by helmoski — APPROVED** [review]

**Pull request merged**